From: NeilBrown <neilb@suse.com>
Subject: Don't restrict NFSv4.2 on openSUSE
Patch-mainline: Never, suse specific
References: bsc#1138719

We default the nfs.max_minor_version to 1 on some SLE kernels,
but don't need to do that for openSUSE - Leap or Tumbleweed

---
 fs/nfs/super.c |    4 ++++
 1 file changed, 4 insertions(+)

--- a/fs/nfs/super.c
+++ b/fs/nfs/super.c
@@ -76,11 +76,15 @@
 #define NFS_DEFAULT_VERSION 2
 #endif
 
+#ifdef CONFIG_SUSE_PRODUCT_SLE
 #if CONFIG_SUSE_VERSION < 15 || (CONFIG_SUSE_VERSION == 15 && CONFIG_SUSE_PATCHLEVEL == 0)
 static int max_minor_version = 1;
 #else
 static int max_minor_version = 2;
 #endif
+#else
+static int max_minor_version = 2;
+#endif
 
 #define NFS_MAX_CONNECTIONS 16
 
